src/pretix/control/forms/event.py

```diff
@@ -471,8 +471,8 @@ class InvoiceSettingsForm(SettingsForm):
 class MailSettingsForm(SettingsForm):
     mail_prefix = forms.CharField(
         label=_("Subject prefix"),
-        help_text=_("This will be prepended to the subject of all outgoing emails. This could be a short form of "
-                    "your event name."),
+        help_text=_("This will be prepended to the subject of all outgoing emails, formatted as [prefix]. "
```

Comment:
Hmmm, yeah, maybe the validator is too much. The JavaScript solution you suggested is also probably overkill (but it's nice, I'd merge it, but I wouldn't do it myself in your place). Maybe just replace the "formatted as [prefix]" with "The subject will be formatted as '[prefix] Subject…'" or something like this.